sdk/sync/client_gc.go

```diff
@@ -0,0 +1,109 @@
+package sync
```

Comment:
as an alternative to cooperative gc, why can't we set a TTL to keys and let them expire after an interval e.g. 5 hours.

Reply:
TTLs are not access-based, they are set explicitly by the client. Mimicking access-based TTLs is possible, but it entails every client resetting the TTL every time they perform an access/write operation on that key.

You _could_ have the _creator_ of the key set a long TTL and hoping the test plan doesn't last longer than that, which would make it impossible to write test workloads that last hours (e.g. testing against natural churn in the live network). It also introduces a timing assumption and limitation, which conceptually is flawed IMO.